app/src/main/java/com/babylon/wallet/android/presentation/settings/accountsecurity/depositguarantees/DepositGuaranteesViewModel.kt

```diff
@@ -28,6 +28,10 @@ class DepositGuaranteesViewModel @Inject constructor(
     }
 
     fun onDepositGuaranteeChanged(depositGuarantee: String) {
+        if (depositGuarantee.contains("-")) {
+            return
+        }
+
```

Comment:
I agree with Micheal here. Better convert this to BigDecimal just to be more safe

Reply:
@micbakos-rdx @giannis-rdx Hmm I kind of disagree here for this specific case even though I think the BigDecimal concept is preferred in general.
So we have three case where input is :
"-" (minus)
"." (dot)
"" (empty string)

All of them return null when you convert string to BigDecimal which means we have to return and not complete function. However I want empty string to be visible there in the field (similarly as ".").
Then we will have to differentiate this but I guess we are making this the same (or more complicated) as it was.

What do you think ?